src/snippets.ts

```diff
@@ -0,0 +1,83 @@
+import {ElementSnippet} from './editor-service';
+
+const snippets: ElementSnippet[] = [
+  {
+    'tagname': 'html-import',
+    'description': 'Template definition of an HTML import.',
+    'prefix': 'hi',
+    'expandTo': '<link rel="import" href="bower_components/.html">',
+    'expandToSnippet': '<link rel="import" href="${1:bower_components}/${2}/${2}.html">'
```

Comment:
This is tricky. If it's an app then they'll want `bower_components/` or similar, but if it's a standalone element it should be `../`.

We really need a signal that lets us tell whether some code is an app or a standalone component.